src/RCController/RCController.cpp

```diff
@@ -80,46 +93,53 @@ namespace tritonai::gkc
 
             if(!_receiver.messageAvailable) continue; // Stop if no message available
 
-            bool is_the_same_data = Map.throttle(busData[ELRS_THROTLE]) == _packet.throttle &&
-                Map.steering(busData[ELRS_STEERING]) == _packet.steering;
-
-            bool is_all_zero = abs(100*Map.throttle(busData[ELRS_THROTLE])) <= 5 && abs(100*Map.throttle(busData[ELRS_STEERING])) <= 5;
-
-            // Check if the values are the same as the previous ones
-            if (is_the_same_data && !is_all_zero)
-                continue; // Stop if the values are the same
+            // std::cout << "Bus data: " << (int)(100*busData[0]) <<
+            //     " " << (int)(100*busData[1]) <<
+            //     " " << (int)(100*busData[2]) <<
+            //     " " << (int)(100*busData[3]) <<
+            //     " " << (int)(100*busData[4]) <<
+            //     " " << (int)(100*busData[5]) <<
+            //     " " << (int)(100*busData[6]) <<
+            //     " " << (int)(100*busData[7]) <<
+            //     " " << (int)(100*busData[8]) <<
+            //     std::endl;
 
             // Stop if the emergency stop is not active
             bool temp_active = Map.is_active(
                 busData[ELRS_EMERGENCY_STOP_LEFT],
                 busData[ELRS_EMERGENCY_STOP_RIGHT]
             );
 
-            if(is_all_zero){
+            if(!temp_active){
                 _packet.throttle = 0.0;
                 _packet.steering = 0.0;
-                _packet.brake = 0.0;
+                _packet.brake = Map.throttle_ratio(busData[ELRS_RATIO_THROTTLE]);
                 _packet.is_active = temp_active;
-                _packet.autonomy_mode = Map.getAutonomyMode(
-                    busData[ELRS_TRI_SWITCH_RIGHT]
-                );
                 _packet.publish(*_sub);
                 continue;
             }
 
-            // if(temp_active == false && _packet.is_active == false) continue; // Stop if the values are the same
+            bool keep_constant_thr = Map.keep_constant_thr(busData[ELRS_HOLD_THROTTLE]);
+            bool is_all_zero = abs(100*Map.throttle(busData[ELRS_THROTLE])) <= 5 && abs(100*Map.throttle(busData[ELRS_STEERING])) <= 5;
```

Comment:
Is the second part of the and suppose to just be a normalize instead of a throttle function? Do you even need the throttle function if it is just calling normalize?

Reply:
The Map returns normalized values, I am checking if the throttle and steering axes are below 5% as a dead zone.

Reply:
I am more talking about `abs(100*Map.throttle(busData[ELRS_STEERING])) <= 5`. Is Map.throttle not equivalent to Map.normalize? Are you planning to add more features later?

Reply:
no, you're correct.